src/Microsoft.DotNet.Cli.Utils/CommandResolver.cs

```diff
@@ -83,7 +84,7 @@ private static ProjectContext GetProjectContext(NuGetFramework framework)
                 return null;
             }
 
-            var projectContext = ProjectContext.Create(projectRootPath, framework);
+            var projectContext = ProjectContext.Create(projectRootPath, framework, PlatformServices.Default.Runtime.GetAllCandidateRuntimeIdentifiers());
```

Comment:
Cannot do on `PlatfromServices` but can `PlatfromServices.Default.GetAllCandidateRuntimeIdentifiers`, don't know if it would help a lot.